hummingbot/connector/exchange/kucoin/kucoin_exchange.pyx

```diff
@@ -766,30 +767,32 @@ cdef class KucoinExchange(ExchangeBase):
                              f"{trading_rule.min_order_size}.")
 
         try:
-            exchange_order_id = await self.place_order(order_id, trading_pair, decimal_amount, False, order_type,
-                                                       decimal_price)
             self.c_start_tracking_order(
                 client_order_id=order_id,
-                exchange_order_id=exchange_order_id,
+                exchange_order_id="",
```

Comment:
By default when the exchange_order_id is still not set it should be None (this can be checked in the code from InFlightOrderBase)

Reply:
Ah yes, fixed (in next push)